README.md

```diff
@@ -4,61 +4,84 @@
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg?style=for-the-badge)](LICENSE) [![Wally](https://img.shields.io/github/v/tag/ukendio/jecs?&style=for-the-badge)](https://wally.run/package/ukendio/jecs)
 
-Just a stupidly fast Entity Component System
+# Jecs - Just a Stupidly Fast ECS
 
--   [Entity Relationships](https://ajmmertens.medium.com/building-games-in-ecs-with-entity-relationships-657275ba2c6c) as first class citizens
--   Iterate 800,000 entities at 60 frames per second
--   Type-safe [Luau](https://luau-lang.org/) API
--   Zero-dependency package
--   Optimized for column-major operations
--   Cache friendly [archetype/SoA](https://ajmmertens.medium.com/building-an-ecs-2-archetypes-and-vectorization-fe21690805f9) storage
--   Rigorously [unit tested](https://github.com/Ukendio/jecs/actions/workflows/ci.yaml) for stability
+A high-performance Entity Component System (ECS) for Roblox games.
 
-### Example
+## Features
+
+- 🚀 **Blazing Fast**: Iterate over 800,000 entities at 60 frames per second
+- 🔗 **Entity Relationships**: First-class support for [entity relationships](docs/learn/concepts/relationships.md)
+- 🔒 **Type Safety**: Fully typed API for both [Luau](https://luau-lang.org/) and TypeScript
+- 📦 **Zero Dependencies**: No external dependencies required
+- ⚡ **Optimized Storage**: Cache-friendly [archetype/SoA](https://ajmmertens.medium.com/building-an-ecs-2-archetypes-and-vectorization-fe21690805f9) storage
+- ✅ **Battle-tested**: Rigorously [unit tested](https://github.com/Ukendio/jecs/actions/workflows/ci.yaml) for stability
+
+## Documentation
+
+- [Getting Started](docs/learn/overview/get-started.md)
+- [API Reference](docs/api/jecs.md)
+- [Concepts](docs/learn/concepts/)
+- [Examples](examples/)
+- [FAQ](docs/learn/faq/common-issues.md)
+
+## Quick Example
 
 ```lua
 local world = jecs.World.new()
 local pair = jecs.pair
 
--- These components and functions are actually already builtin
--- but have been illustrated for demonstration purposes
-local ChildOf = world:component()
-local Name = world:component()
+-- Define components
+local Position = world:component() :: jecs.Entity<Vector3>
+local Velocity = world:component() :: jecs.Entity<Vector3>
```

Comment:
I would use `jecs.Id` to represent component types. They are more terse and actually a bit more correct.